src/test/resources/testng.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="MyTestSuite">
+    <listeners>
+        <listener class-name="com.solvd.MyListener"/>
+    </listeners>
+    <test name="TestChrome" parallel="tests" thread-count="4">
+        <parameter name="browser" value="chrome"/>
+        <classes>
+            <class name="com.solvd.HomeTest"/>
+            <class name="com.solvd.LoginTest"/>
+            <class name="com.solvd.CartTest"/>
+        </classes>
+    </test>
+    <test name="TestFirefox" parallel="methods" thread-count="4">
```

Comment:
parallel="methods" inside FirefoxTests is not recommended unless your tests are designed to run methods in parallel within the same driver session.

Reply:
If you do want full parallel method execution, use parallel="methods" at the suite level and ensure your code is 100% thread-safe